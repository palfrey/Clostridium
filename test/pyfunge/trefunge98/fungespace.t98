v
># ha"derongi h :DAB">:#,_@_,#! #:<"BAD: h reflects"a
   >a"enalp tsrif setirwrevo enalp dnoces :DAB">:#,_@

   ^
   >a"dengilasim era senalp :DAB">:#,_@
   >a"tcerroc si ecaps egnuF :DOOG">:#,_@

