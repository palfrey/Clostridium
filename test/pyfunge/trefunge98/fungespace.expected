GOOD: Funge space is correct

